dict/P/pandunia.tsv

```diff
@@ -455,6 +457,8 @@ PWN:committee.n.01		komisia	ko.mis·ia
 PWN:common.a.01		konhave	kon·hav'e	
 PWN:companion.n.01		sahib	sahib	ara: صَاحِب (ṣāḥib), fas: صاحب (sāhib), hin:साहिब (sāhib), ben:সাহেব (śaheb), may:sahabat, swa:sahibu
 PWN:compare.v.03		kompar	kom.par	
+PWN:compass.n.01		busola	busola	fra:boussole, spa:brújula, por:bússola, ara:بُوصُلَة (būṣula), tur:pusula, rus:буссо́ль (bussól’)
+PWN:compass.n.04		pergar	pergar	fas:پَرْگار (pargâr), tur:pergel, ara:بِرْكَار (birkār), swa:bikari

```

Comment:
The Persian original seems to mean 'circle maker' (_pari_- + -_kâr_). So in Pandunia _sirke grafer_?